server/utils/helpers.js

```diff
@@ -1,3 +1,4 @@
+import '@babel/polyfill';
```

Comment:
This import is unnecessary

Reply:
Move this to your entrypoint (`index.js`)

Reply:
I have moved it